gulp-core-build-sass/src/SassTask.ts

```diff
@@ -76,7 +78,13 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
 
     if (this.taskConfig.useCSSModules) {
       this.logVerbose('Generating css modules.');
-      return this._processFiles(gulp, srcPattern, completeCallback, modulePostCssPlugins);
+      return this._processFiles(gulp, srcPattern, completeCallback, modulePostCssPlugins,
+        (file: gulpUtil.File) => {
+          if (!path.basename(file.path).match(/module\.css$/)) {
```

Comment:
Isnt this match wrong? Shouldnt it be scss?

But also why? Why would this filename format matter? Do we treat scss vs module.scss files differently?